### PR TITLE
Terminology update: High-availability control plane

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/14964

This PR aligns the naming of the feature `high-availability control plane` (formerly called `high-availability master nodes`). Also in the feature page the term `master node` is replaced with `control plane node`. Links and shortcodes are adapted accordingly.

### CODE REVIEW

**Comment by marians** [comment]
FYI @alex-dabija as the page and feature is owned by Firecracker.

**Comment by ubergesundheit** [comment]
I am a little confused. Isn't what we called `control plane` now `management cluster` ?

**Review by alex-dabija — APPROVED** [review]

File: src/content/advanced/high-availability/control-plane/index.md [hunk 1]
```diff
@@ -1,75 +1,78 @@
 ---
-linkTitle: High-availability masters
-title: High-availability Kubernetes masters
-description: A general description of high availability of masters as a concept, it's benefits, and some details you should be aware of.
+linkTitle: High-availability control plane
+title: High-availability Kubernetes control plane
+description: For production clusters with high availability requirements, Giant Swarm on AWS enables control planes with three control plane nodes and three etcd replicas spread over multiple availability zones. Here we explain most details.
```
Comment (stone-z) [inline src/content/advanced/high-availability/control-plane/index.md:4]: I thought about alternative wordings but think it just sounds cleanest without. If it's important to have a quasi-CTA there, I'd suggest "Details here"/"Details inside" or moving it to the beginning of the sentence: "For production clusters with high availability requirements, this article covers configuring clusters with three control plane nodes and `etcd` replicas spread over multiple availability zones".
```suggestion
description: For production clusters with high availability requirements, Giant Swarm on AWS enables control planes with three control plane nodes and three etcd replicas spread over multiple availability zones.
```

**Review by stone-z — APPROVED** [review]
It occurs to me that the intention might be to do another pass to align control plane naming? If so feel free to ignore the contextual updates around the links and get them in the next pass.

**Comment by marians** [comment]
@stone-z This wasn't supposed to be the only PR for the terminology change. I wanted to keep it digestible and easier to review by making several PRs.

**Pull request merged**